pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +116,127 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	repoNamesStr := func(repoCount int) string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, fmt.Sprintf("%s%d", repoNameInitial, i))
+		}
+		jsonStr, err := json.Marshal(repoNames)
+		if err != nil {
+			return "[]"
+		}
+		return fmt.Sprint(string(jsonStr))
```

Comment:
Why do we need to convert string to string, then format it? Why can't we just `return jsonStr`?

Reply:
mmm, yeah... I don't get what you're doing here

Reply:
fixed